src/App/Fossa/Main.hs

```diff
@@ -107,6 +110,31 @@ experimentalLicenseScanCommand = command "experimental-license-scan" (info runIn
     runInit :: Parser (IO ())
     runInit = pure $ putStrLn "The 'experimental-license-scan' has been deprecated and renamed to 'license-scan'. Please use the 'license-scan' command instead."
 
+feedbackCommand :: Mod CommandFields (IO ())
+feedbackCommand = command "feedback" (info feedbackPrompt $ progDescDoc $ formatStringToDoc "Provide feedback on your fossa-cli experience, submit feature requests, and report bugs/issues")
+  where
+    feedbackPrompt :: Parser (IO ())
+    feedbackPrompt =
+      pure $
+        putDoc $
+          vsep
+            [ newlinePreceding "At FOSSA, we are committed to delivering an exceptional user experience and are continously working towards improving our product."
+            , "Your feedback is crucial in shaping our ongoing efforts to innovate and provide an even better user experience!"
+            , ""
+            , "* Report bugs and issues at:"
+            , newlineTrailing . newlinePreceding $ indent 4 $ pretty supportUrl
+            , "* Submit feature requests to:"
+            , newlineTrailing . newlinePreceding $ indent 4 "support@fossa.com"
```

Comment:
The support site basically just has them send an email, should we join these two bullets?

Reply:
I think having the support site is nice in the sense that the default ticket form already gives details on what to provide to the support team.

i.e. debug bundle, additional photos , etc

It also seems more structured than a standard email

Reply:
I agree; in that case wouldn't we just always send them to the support site?

Reply:
The current ticket forms look like they're more geared for bug reporting. So If we were to consolidate and send users to the support site for both feature requests and bug reporting, then I think it would be best to create a new feature request ticket template. I would need to look into this as I'm not sure how to do that as of rn . 

<img width="656" alt="Screenshot 2024-02-05 at 1 37 02 PM" src="https://github.com/fossas/fossa-cli/assets/28590628/456751a6-29de-4d19-a9a6-f0ae9eb144e2">

Reply:
Hmm, okay. Let's ask Brian which he prefers. I'm good with whatever you and he work out.